src/core/ext/transport/chttp2/transport/chttp2_transport.cc

```diff
@@ -165,6 +165,16 @@ static void reset_byte_stream(void* arg, grpc_error* error);
 // GRPC_EXPERIMENTAL_DISABLE_FLOW_CONTROL
 bool g_flow_control_enabled = true;
 
+namespace grpc_core {
+TestOnlyGlobalHttp2TransportInitCallback test_only_init_callback = nullptr;
+
+void TestOnlySetGlobalHttp2TransportInitCallback(
```

Comment:
I'd suggest having a separate shutdown callback, instead of reusing the init callback on destruction.

Reply:
Done!